services/web/server/docker/entrypoint.sh

```diff
@@ -8,6 +8,13 @@ INFO="INFO: [$(basename "$0")] "
 WARNING="WARNING: [$(basename "$0")] "
 ERROR="ERROR: [$(basename "$0")] "
 
+# Read self-signed SSH certificates (if applicable)
+#
+# In case the service must access a docker registry in a secure way using
+# non-standard certificates (e.g. such as self-signed certificates), this call is needed.
+# It needs to be executed as root.
+update-ca-certificates
```

Comment:
Q: why the web-server needs this? AFAIK it does not access the docker registry directly

Reply:
the comment was copy pasted sorry, let me adjust it